examples/Microsoft.Spark.CSharp.Examples/Microsoft.Spark.CSharp.Examples.csproj

```diff
@@ -13,4 +13,9 @@
     <ProjectReference Include="..\..\src\csharp\Microsoft.Spark\Microsoft.Spark.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.3.1" />
```

Comment:
this shouldn't be required.   `Microsoft.ML.FastTree` brings in this library as a dependency.

Reply:
Oh that's right! Just changed.